docs/assets/templates/services/barebones-.dockstore.yml

```diff
@@ -0,0 +1,23 @@
+version: 1.2
+service:
+  subclass: <DOCKER_COMPOSE | KUBERNETES | HELM | SWARM | NOT_APPLICABLE>
+  name: <String>
```

Comment:
without looking at where this is linked to, this isn't quite `barebones` since name, author, publish, are optional. 
Probably a bad file name, so probably ignore

Reply:
I suggest all of our template examples are 'recommended' or 'best practice' rather than minimal to encourage better metadata.

Reply:
Not sure where to put this comment, but can we also order our template docs in the sidebar: workflows, tools, and then services? I think having services first is weird since it is beta.